build-system/server/app.js

```diff
@@ -1567,4 +1568,10 @@ app.use('(/dist)?/rtv/*/v0/analytics-vendors/:vendor.json', (req, res) => {
     });
 });
 
+// Used by test/unit/test-3p.js to test script loading.
```

Comment:
Can you point to where? What's using `require`?

Reply:
This merely mimics the kind of script that Karma serves by default when you fetch a source file from the server. The only test that currently relies on the fact that we're serving all test JS files is this one:

https://github.com/ampproject/amphtml/blob/b1a8300ac540823c097b58b30152a6dfb1bec494/test/unit/test-3p.js#L294-L305

With the change to the unified JS file, we are no longer serving this JS file, so I had to introduce this alternative. (I tried a way to serve but not include all source JS files in the Karma server, but that makes our tests fail.)